.github/ALLOWLIST

```diff
@@ -25,3 +25,4 @@ internal/common
 
 ## UNMAINTAINED components
 exporter/skywalkingexporter/
+receiver/googlecloudspannerreceiver/
```

Comment:
```suggestion
```
This looks unrelated to this PR, we should probably open a separate PR if necessary. Same goes for all other references to the `googlecloudspannerreceiver`

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33385